docs/self-managed/identity/getting-started/install-identity.md

```diff
@@ -1,18 +1,20 @@
 ---
 id: install-identity
-title: "Installation and first steps"
-sidebar_label: "Installation and first steps"
-description: "Learn more about installing Identity, accessing the UI, default users, the home screen, and more."
+title: "First steps"
```

Comment:
This is only an introduction to initial configuration and UIs after logging in with demo demo. All installation instructions were moved.